bin/ci/run-tests.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+set -u
```

Comment:
Good call, thanks for the guidance.

By the way, I was suspicious this wouldn't work with BATs, since it's just bash and failing tests would count as errors, but I tested it and it seems to not get killed by failing tests OK

```
1..3
ok 1 Version matches
not ok 2 This fails
# (in test file ant/tests/test.bats, line 8)
#   `[[ 1 == 2 ]]' failed
ok 3 Version matches (again)
```